Headers/DebugServer2/Target/Linux/Thread.h

```diff
@@ -31,6 +31,9 @@ class Thread : public ds2::Target::POSIX::Thread {
 #endif
 
 protected:
+  void fillStoppointData();
```

Comment:
Nit: the name of this function is not the best. Given that it sets the stop reason to `Breakpoint` if it doesn't find a corresponding watchpoint, it does more than `Stoppoint` data.